hack/check_defaults.sh

```diff
@@ -111,55 +103,6 @@ for JPATH in "${LMPATHS[@]}"; do
     sleep 2
 done
 
-echo "Check that permittedHostDevices defaults are behaving as expected"
-
-./hack/retry.sh 10 3 "${KUBECTL_BINARY} patch hco -n \"${INSTALLED_NAMESPACE}\" --type=json kubevirt-hyperconverged -p '[{ \"op\": \"replace\", \"path\": /spec, \"value\": {} }]'"
-for JPATH in "${PERMITTED_HOST_DEVICES_PATHS[@]}"; do
-    ./hack/retry.sh 10 3 "${KUBECTL_BINARY} patch hco -n \"${INSTALLED_NAMESPACE}\" --type='json' kubevirt-hyperconverged -p '[{ \"op\": \"remove\", \"path\": '\"${JPATH}\"' }]'"
-    PHD=$(${KUBECTL_BINARY} get hco -n "${INSTALLED_NAMESPACE}" kubevirt-hyperconverged -o jsonpath='{.spec.permittedHostDevices}')
-    if ! echo "${PHD}" | grep "${PERMITTED_HOST_DEVICES_DEFAULT1}"; then
```

Comment:
I think we should keep the test logic but removing PERMITTED_HOST_DEVICES_DEFAULT1 and PERMITTED_HOST_DEVICES_DEFAULT2 on line 27, 28 ensuring we get empty defaults.

Reply:
But there are no default devices, so there is nothing in the CR.